app/javascript/components/request-workflow-status/data.js

```diff
@@ -59,6 +59,15 @@ export const workflowStatusData = (response) => {
     return undefined;
   }
   const rows = response.context ? rowData(response.context) : [];
+  if (response.context && response.context.State) {
+    rows.push({
+      id: `${response.context.State.Guid.toString()}+${Math.random()}`,
```

Comment:
@GilbertCherrie why the Math.random() suffix?  Maybe related to another comment, we only want to append this if it doesn't exist in the StateHistory (if a workflow is complete, the last state will also be in StateHistory)

Reply:
Maybe when a workflow is complete we should null out the `context["State"]` ?  @Fryguy wdyt

Reply:
but regardless, the random number doesn't make sense to me.

Reply:
Yeah that was just due to a duplicate state causing react warnings of duplicate ids